### PR TITLE
Company profile edit view - improvements

### PR DESCRIPTION
**Pull request by monikasulik**
Just a few minor improvements and a test fix for #27

### CODE REVIEW

File: registration/forms.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class CompanyForm(forms.Form):
 class CompanyBasicInfoForm(forms.Form):
     company_name = forms.CharField()
```
Comment (monikasulik) [inline registration/forms.py]: Possibly all these fields should be limited to 255? Feedback welcome.
Also should I be explicitly adding labels? Found something about that in our Development Practices on Confluence.

Comment (richtier) [inline registration/forms.py]: eventually some fields will be defined in a directory-api model. As part of that ticket we will determined the max_length. When that is done we can update these forms as a follow up task.

File: registration/forms.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ class CompanyForm(forms.Form):
...
     website = forms.URLField()
-    description = forms.CharField()
+    description = forms.CharField(widget=forms.Textarea)
```
Comment (richtier) [inline registration/forms.py]: the object needs to be instantiated. 
```py
password = forms.CharField(widget=forms.PasswordInput())
```

File: registration/views.py [hunk 3]
```diff
@@ -76,4 +76,6 @@ def get_template_names(self):
         ]
 
     def done(self, form_list, form_dict):
+        # TODO: ED-144
+        # Change this to redirect to the company profile detail page
```
Comment (richtier) [inline registration/views.py]: :+1:

File: registration/forms.py (hunk 2 above)
Comment (monikasulik) [inline registration/forms.py]: It actually works as is and it's how the 1.10 docs have it: https://docs.djangoproject.com/en/1.10/ref/forms/widgets/#specifying-widgets
I can change it if you prefer, but don't think it has any advantage?

File: registration/forms.py (hunk 1 above)
Comment (monikasulik) [inline registration/forms.py]: Added a TODO comment  :)

File: registration/forms.py (hunk 2 above)
Comment (richtier) [inline registration/forms.py]: no need to change, looks like instantiating the widget is legacy. Lets use the way you did it.

**Pull request merged**